README.md

```diff
@@ -3,25 +3,44 @@
 ## Objectives
 + Use Ajax `get` method
 + Use `success` callback
-+ USe `error` callback
++ Use `error` callback
 
 ## Intro
+Let's test out our new Ajax skills by making a "Mad Lib" Maker. [Mad Libs](http://www.madlibs.com/) are paragraphs of text where certain verbs and nouns are missing. It's the job of the reader to pick random words to fill these gaps. The result is usually riduculous and makes for a good laugh.
 
-"Mad Lib" Maker
+For our Mad Lib maker, we will have a static sentence that will include blanks for one missing noun and one missing verb. The HTML page will also have two associated buttons, one for nouns and one for verbs.  When the user clicks the button, we will load words from an external HTML file using ajax and randomly select a word to add to the sentence.
 
-have files `noun.html` and `verb.html` which contain lists of random nouns and verbs respectively.
+Hopefully when we are done we will have made the world a better place.
 
-index.html should have a fill in the blank structure. 
-
-
-```html
-<p> Today i went to the</p> <p class="noun"></p>.<p> while i was there i decided to</p><p class="verb"></p>
-.....
-```
-
-index.html should have a button for each noun and each verb - click events attached to the buttons that take a random noun from noun.html and a random verb from verb.html and append to `index.html` to fill in the madlib
+Note: It's recommended you run ```python -m SimpleHTTPServer``` in your lab folder to start a simple server to host your files.
 
 ## Instructions
-
-Needs tests
-
+All of the HTML and javascript files have been provided. The file ```index.html``` has the mad lib UI. You will be adding your code to ```script.js```. There are also Jasemine tests under ```specs``` that you can run by loading the ```SpecRunner.html``` file in Firefox to test your work.
+
+### A. Random Noun Button
+For all spans with class ```noun``` a randomly selected word from ```noun.html``` will be inserted.
+
+1. Bind a click event to the noun button.
```

Comment:
Can you change your instructions to be more "feature"-y rather than super prescriptive? It makes it bit too easy for the students! So For this I'd say:

 - When a user clicks on the noun button, load `noun.html` and radomly select a noun from the `span`
 - Replace the in your index with the noun your selected